apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -87,6 +87,8 @@ var ApplabVisualizationColumn = React.createClass({
       <Visualization key="1"/>,
       this.props.isIframeEmbed && !this.props.isRunning && <IframeOverlay key="2"/>
     ];
+    // Share view still uses image for phone frame. Would eventually like it to
+    // use same code
```

Comment:
One challenge here will be that PhoneFrame is currently app lab specific, and we use the phone frame for share pages on multiple apps. Might be we'll need a new more generic phone frame component.